src/screens/Communities/index.tsx

```diff
@@ -91,7 +92,6 @@ const CommunitiesScreen: VFC = () => {
 
   const handleLoadMore = () => {
     if (nextPage) {
-      setLoading(LoadingState.IS_LOADING_MORE);
       onQueryCommunities({ page: nextPage });
     }
   };
```

Comment:
We can extract this logic to a hook `usePagination`? Check it out here: https://github.com/EkoCommunications/AmityConsoleWeb/blob/develop/src/hooks/usePagination.js